internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -205,7 +205,7 @@ func (r *Reconciler) reconcileMonitoringSecret(
 	intent.Data = make(map[string][]byte)
 
 	if len(existing.Data["password"]) == 0 || len(existing.Data["verifier"]) == 0 {
-		password, err := util.GeneratePassword(util.DefaultGeneratedPasswordLength)
+		password, err := util.GenerateASCIIPassword(32)
```

Comment:
:thought_balloon:  Would we ever want to use  the `AlphaNumeric` password option here? I don't believe this password would be used anywhere else, unlike the Postgres password, so I think this makes sense.

Reply:
Maybe if something changes in the exporter or how it is configured. We've managed with an ASCII password so far.

Reply:
Sure, if we don't have a clear case, I'm fine for leaving this as is for now.